forge/lang/expander.rkt

```diff
@@ -289,6 +290,29 @@
               (~optional name:NameClass)
               test-block:TestBlockClass)))
 
+;; Sidd
+
+
+  (define-syntax-class TestConstructClass
+    (pattern decl:ExampleDeclClass)
+    (pattern decl:TestExpectDeclClass)
+    (pattern decl:PropertyWhereDeclClass))
```

Comment:
Comment-out this last pattern for now. No nesting.

Reply:
From what i can tell, nesting will be needed for every assignment after Forge 2. Are we sure we want to remove it?

Reply:
That's surprising. We better discuss with Tim. Where are examples?

Reply:
There is one in the is_undirected_tree example. We can get rid of it now, but I propose we do it in the following way:

- This PR goes into dev
- We then have a subsequent PR removing nested property-wheres. We could then revert this PR if we ever needed to enable the feature without having to have commented-out code.

Reply:
Sure. It sounds like overengineering to me, but that's fine.